commands/auth/login/login.go

```diff
@@ -125,14 +125,15 @@ func loginRun() error {
 
 	hostname := opts.Hostname
 	apiHostname := opts.Hostname
+	defaultHostname := glinstance.Default()
```

Comment:
Use the [OverridableDefault()](https://github.com/profclems/glab/blob/c608850edb768d2a9be2d98874440a3643a32a42/internal/glinstance/host.go#L28) instead of [Default()](https://github.com/profclems/glab/blob/c608850edb768d2a9be2d98874440a3643a32a42/internal/glinstance/host.go#L18). 
Default() returns the value in the [`defaultHostname`](https://github.com/profclems/glab/blob/c608850edb768d2a9be2d98874440a3643a32a42/internal/glinstance/host.go#L10) which is constant. The value is `gitlab.com`.
OverridableDefault() will check if the value has been overriden and returns it, if not, returns the default.
```suggestion
	defaultHostname := glinstance.OverridableDefault()
```

Reply:
Oops this has been resolved in the other commit